FakeXrmEasy.Shared/FakeXrmEasy.Shared.projitems

```diff
@@ -85,4 +86,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)XrmOrderByAttributeComparer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)XrmRealContext.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="$(MSBuildThisFileDirectory)FakeMessageExecutors\NewFolder\" />
```

Comment:
I guess this is not intended :)

Reply:
Oops! New to Git - how do I deal with this? Delete it locally and then....?

Reply:
Just remove it, commit + push

Reply:
Thanks - sorted now...

Reply:
Cheers guys